addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -643,6 +671,12 @@ function factory(dependencies) {
             isCausal: true,
             inverse: 'threadViewTopbarOwner',
         }),
+        /**
+         * Determines whether this thread description is editable.
```

Comment:
in this case `Determines` should be `States` instead: changing the value does not control the rights to edit, it's just an indicative value.